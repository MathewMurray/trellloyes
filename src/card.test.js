import Card from './card';
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

describe('smoke test and snapshot test',() =>{
it('generates card with title and content',()=>{
const div = document.createElement('div');
ReactDOM.render(<Card />,div)
ReactDOM.unmountComponentAtNode(div);
});

it('renders to the ui',() =>{
    const tree = renderer 
    .create(<Card title = "test card" content = "test content"/>)
    .toJSON();
    expect(tree).toMatchSnapshot();
});
});