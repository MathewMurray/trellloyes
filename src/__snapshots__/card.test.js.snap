// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke test and snapshot test renders to the ui 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    test card
  </h3>
  <p>
    test content
  </p>
</div>
`;
